skfem/assembly/dofs.py

```diff
@@ -66,3 +66,22 @@ def all(self, key: Union[List[str], str] = None):
                           "Returning an empty set of DOF's.")
 
         return output
+
+    @staticmethod
+    def _merge_dicts(d1, d2):
+        keys = set(d1).union(d2)
+        no = np.array([])
+        return dict((k, np.union1d(d1.get(k, no), d2.get(k, no)))
```

Comment:
We don't need to sort the merged degrees of freedom here do we?  I don't think so, just checking.

Reply:
Do you mean that something else than `union1d` would suffice?

Reply:
No, I worried for a moment that leaving the degrees of freedom unsorted here might be a problem.  I don't think it is, just checking.